.travis.yml

```diff
@@ -1,6 +1,8 @@
+dist: xenial
 language: python
-services:
-  - docker
+# Docker deployment is now disabled
+# services:
+#   - docker
```

Comment:
@anadahz do you have a sense of what we were using Docker for?

Reply:
The docker machine on demo.probe.ooni.io.

Reply:
Lol, then please just remove the Docker section. We don't need it anymore AFAIK.